python/cudf/cudf/core/df_protocol.py

```diff
@@ -716,7 +716,7 @@ def _protocol_to_cudf_column_numeric(
     _dbuffer, _ddtype = buffers["data"]
     _check_buffer_is_on_gpu(_dbuffer)
     cudfcol_num = build_column(
-        Buffer(_dbuffer.ptr, _dbuffer.bufsize),
+        Buffer(data=_dbuffer.ptr, size=_dbuffer.bufsize, owner=None),
```

Comment:
Is it deliberate that you're not calling `as_device_buffer_like` in this file?

Reply:
`as_device_buffer_like()` only accept one argument now.

Reply:
Ah, makes sense (though that means that dataframes sent through the `__dataframe__` protocol will not pick up `SpillableBuffer`s when those appear).

Reply:
Right, we might have to handle this in the spill manager PR if we can determent that the buffer is spillable.